### PR TITLE
add training part 1

### PR DESCRIPTION
**Pull request by afik**
Screen Shot 2021-05-02 at 14 59 31

### CODE REVIEW

File: echo/server.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
```
Comment (bxcodec) [inline echo/server.go]: nit: can we use the V4?

Comment (weisurya) [reply echo/server.go]: +1 @afik 

to give a better context, there is a breaking changes between v1 and v4. So far, AFAIK if you use v1 component, you couldn't use v4 component and vice versa :sob:

File: test.json [hunk 2]
```diff
@@ -0,0 +1 @@
...ayment_code \": \"abc123\" \r\n}","options":{"raw":{"language":"json"}}},"url":"http://{{golang-training-url}}/inquiry"},"response":[]},{"name":"POST Payment","event":[{"listen":"test","script":{"type":"text/javascript","exec":["const resBody = JSON.parse(responseBody);\r","const reqBody = JSON.parse(pm.request.body.raw);\r","\r","pm.test(\"Should create new payment code and return success request (200)\", function () {\r","    pm.expect(resBody.transaction_id).to.be.equal(reqBody.transaction_id);\r","    pm.expect(resBody.name).to.be.equal(reqBody.name);\r","    pm.expect(resBody.amount).to.be.equal(reqBody.amount);\r","    pm.expect(resBody.status).to.be.equal('SUCCESS');\r","    pm.response.to.have.status(200);\r","});\r",""]}}],"id":"62f1276a-e885-4bd8-bf3a-edce611f57e0","request":{"method":"POST","header":[],"body":{"mode":"raw","raw":"{\r\n    \"transaction_id\": \"{{part_5_transaction_id}}\",\r\n    \"amount \": {{part_5_amount}}, \r\n    \"name\": \"{{part_5_name}}\", \r\n    \"payment_code\": \"{{part_5_payment_code}}\"\r\n}","options":{"raw":{"language":"json"}}},"url":"http://{{golang-training-url}}/payment"},"response":[]}],"id":"db2ff90e-cd2b-4e72-b396-6c3739dc1326"}]}
```
Comment (weisurya) [inline test.json:1]: Good practice of putting postman test in a service!

**Review by weisurya — CHANGES_REQUESTED** [review]

File: echo/server.go (hunk 1 above)
Comment (afik) [reply echo/server.go]: Hoo, thankss. That actually was added by the IDE, didn't realized it's different version hehe

**Pull request merged**